server/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "express": "^4.17.1",
     "uniqid": "^5.2.0",
+    "unique-names-generator": "^4.2.0",
```

Comment:
Nice one!

Reply:
I liked it so much that you can't use your own name. Just reroll and get a new one